src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/FindAllReferencesHandler.cs

```diff
@@ -105,16 +120,21 @@ internal async override Task<VSReferenceItem[]> HandleRequestAsync(ReferencePara
                 PartialResultToken = token // request.PartialResultToken
             };
 
+            _logger.LogInformation("Attaching to progress listener.");
+
             if (!_lspProgressListener.TryListenForProgress(
                 token,
                 onProgressNotifyAsync: (value, ct) => ProcessReferenceItemsAsync(value, request.PartialResultToken, ct),
                 DelayAfterLastNotifyAsync,
                 cancellationToken,
                 out var onCompleted))
             {
+                _logger.LogWarning("Failed to attach to progress listener.");
                 return null;
             }
 
+            _logger.LogInformation($"Requesting FAR for {projectionResult.Uri}.");
```

Comment:
Personally I'd avoid Acronyms like "FAR". It won't mean anything to a user or to someone outside of the Razor team, so "Find All References" would be more clear.